pyatv/protocols/companion/auth.py

```diff
@@ -158,7 +158,10 @@ async def verify_credentials(self) -> bool:
             },
         )
 
-        # TODO: check status code
+        # check status code
+        if resp['_pd'][2] != 4:
```

Comment:
I'm a bit uncertain what this check really is? Isn't `2` here the same as https://github.com/postlund/pyatv/blob/8d78f516afe0e738d5eedc054a22fc27b6900d0c/pyatv/auth/hap_tlv8.py#L18 ?